coder-observability/values.yaml

```diff
@@ -146,6 +149,14 @@ global:
     # global.dashboards.queryTimeout -- how long until a query in Grafana will timeout after
     queryTimeout: 900
 
+runbook:
```

Comment:
Can you make this `runbookViewer` please?
Curious why you made the container names configurable too, that might cause issues.

Reply:
can do - i made them configurable as the hardcoded container names were a bit unclear to me, especially the `madness` one.

i can remove this change from the PR in the meantime while we continue to iterate on the chart.